contracts/PeeranhaContent.sol

```diff
@@ -181,6 +181,48 @@ contract PeeranhaContent is IPeeranhaContent, Initializable, NativeMetaTransacti
         documentationTree.updateDocumentationTree(posts, _msgSender(), communityId, documentationTreeIpfsHash);
     }
 
+    /**
+     * @dev create documentation post.
+     *
+     * Requirements:
+     *
+     * - must be a new documentation post.
+     * - must be a community.
+     * - must be tags.
+    */
+    function createDocumentationPost(uint32 communityId, bytes32 ipfsHash, bytes32 documentationTreeIpfsHash, PostLib.PostType postType, uint8[] memory tags) external {
+        posts.createPost(_msgSender(), communityId, ipfsHash, postType, tags);
+        documentationTree.updateDocumentationTree(posts, _msgSender(), communityId, documentationTreeIpfsHash);
+    }
+
+    /**
+     * @dev Edit documentation post info.
+     *
+     * Requirements:
+     *
+     * - must be a documentation post.
+     * - must be new info about documentation post
+     * - must be a community.
+     * - must be tags
+    */
+    function editDocumentationPost(uint256 postId, bytes32 ipfsHash, bytes32 documentationTreeIpfsHash, uint8[] memory tags) external {
+        posts.editPost(documentationTree, _msgSender(), postId, ipfsHash, tags);
+        documentationTree.editDocumentationTree(posts, _msgSender(), postId, documentationTreeIpfsHash);
+    }
+
+    /**
+     * @dev delete documentation post.
+     *
+     * Requirements:
+     *
+     * - must be a documentation post.
+    */
+    function deleteDocumentationPost(uint256 postId) external {
```

Comment:
This is incorrect.
This action is not to delete the documentation tree completely. It is to delete item from documentation tree.
Need to pass `bytes32 documentationTreeIpfsHash` in that action and the update documentation tree instead of deleting it.